app/api/monitoring.py

```
@@ -0,0 +1,31 @@
+
+@router.post("/daily-report", summary="Trigger Daily Discord Report")
+async def trigger_daily_report(
+    x_monitoring_token: str = Header(None, alias="X-Monitoring-Token")
+):
+    """
+    GCP Cloud Scheduler 등 외부 인프라에서 매일 정해진 시간에 호출할 엔드포인트입니다.
+    이 엔드포인트는 Prometheus 지표를 정리하여 Discord로 데일리 리포트를 전송합니다.
+    """
+    # 보안: 시크릿 토큰이 설정된 경우 헤더 검증 수행
+    if MONITORING_SECRET_TOKEN and x_monitoring_token != MONITORING_SECRET_TOKEN:
+        logger.warning(f"Unauthorized monitoring trigger attempt from IP. Token mismatch.")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)